diff '--exclude=.git' -urN a/meson.build b/meson.build
--- a/meson.build	2020-09-11 08:06:08.000000000 -0500
+++ b/meson.build	2020-10-04 06:29:26.604219801 -0500
@@ -54,7 +54,8 @@
   dependency('gobject-introspection-1.0'),
 ]
 
-gspell_dep = dependency('gspell-1', version: '>= 1.0')
+with_gspell = get_option('spell')
+gspell_dep = dependency('gspell-1', version: '>= 1.0', required: false)
 python3 = python.find_installation('python3')
 
 # Configurations
@@ -94,3 +95,4 @@
 summary('API documentation', get_option('gtk_doc'))
 summary('User documentation', get_option('user_documentation'))
 summary('Require all tests', get_option('require_all_tests'))
+summary('Spell', with_gspell)
diff '--exclude=.git' -urN a/meson_options.txt b/meson_options.txt
--- a/meson_options.txt	2020-09-11 08:06:08.000000000 -0500
+++ b/meson_options.txt	2020-10-04 06:24:56.885457807 -0500
@@ -22,3 +22,9 @@
 )
 
 option('plugin_externaltools', type: 'boolean', value: true)
+
+option(
+  'spell',
+  type:'boolean', value: true,
+  description: 'Build spell support'
+)
diff '--exclude=.git' -urN a/plugins/meson.build b/plugins/meson.build
--- a/plugins/meson.build	2020-09-11 08:06:08.000000000 -0500
+++ b/plugins/meson.build	2020-10-04 06:22:49.884044231 -0500
@@ -23,7 +23,9 @@
 subdir('quickopen')
 subdir('snippets')
 subdir('sort')
-subdir('spell')
+if with_gspell
+  subdir('spell')
+endif
 subdir('time')
 
 if get_option('plugin_externaltools')
